docker-compose.env

```diff
@@ -1,3 +1,6 @@
+######### GO #########
+GOPATH=/go
```

Comment:
Not sure if we need this because the container already sets the `GOPATH` to `/go`, see:

```bash
$ docker run --rm -it golang:1.14-buster env
```

I suppose that an argument for having it would be that if we changed the base image, it would prevent the need for updating the `docker-compose` files, but its the same in `golang:alpine` as well so I don't think we need to set it.